content/en/learn/get-started/installation.mdx

```diff
@@ -39,22 +39,21 @@ import NvidiaIcon from '../../../../app/_components/icons/ihv/nvidia.jsx'
 <Steps>
 ### Install Windows App SDK
 
-| Chrome Version | Edge Version | Known compatible Windows App SDK Version |
+| Chrome Version | Edge Version | Known compatible [Windows App SDK](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads) Version |
```

Comment:
Do we need this link? Will some one install the latest one?
<img width="808" height="481" alt="image" src="https://github.com/user-attachments/assets/b6df7326-c979-4b32-b56e-872f36a98579" />